#! /bin/sh /usr/share/dpatch/dpatch-run
## 01manifest.dpatch by  <az@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: upstream #25403: manifests vs. archive-dir

@DPATCH@
diff -urNad duplicity-0.5.06~/src/gpg.py duplicity-0.5.06/src/gpg.py
--- duplicity-0.5.06~/src/gpg.py	2009-01-10 00:05:11.000000000 +1000
+++ duplicity-0.5.06/src/gpg.py	2009-01-31 14:27:58.717319718 +1000
@@ -158,11 +158,7 @@
             self.gpg_output.close()
             if self.status_fp:
                 self.set_signature()
-            try:
-                self.gpg_process.wait()
-            except IOError, message:
-                if message.args[0] != "GnuPG exited non-zero, with code 131072":
-                    raise
+            self.gpg_process.wait()
         if log.getverbosity() >= 5:
             self.print_log()
         self.logger_fp.close()
